### DISCUSSION TITLE
迷你超級電腦！💻

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#105] [General]
**新聞大綱:**
隨著機器人技術和嵌入式運算改變了重要的經濟領域，對於運算的需求亦持續遽增。為實現更強大的**邊緣運算**，滿足機器人、自主機器及醫療器材等應用，NVIDIA 宣布推出全球體積最小、效能更佳且更節能的**AI 超級電腦**「NVIDIA Jetson AGX Orin」。
採用 NVIDIA Ampere 架構，使其效能為前一代產品（Jetson AGX Xavier）的六倍，每秒可進行 200 兆次的運算，相當於搭載 GPU 伺服器的能力，尺寸只有手掌大小。
使用者可透過 NVIDIA CUDA-X 加速運算堆疊、NVIDIA JetPack 軟體開發套件和最新的 NVIDIA 工具來開發及最佳化應用程式，包括雲端原生開發工作流程。NVIDIA NGC 項目中預先完成訓練的模型已經過最佳化調整，能夠透過 NVIDIA TAO Tookit 和客戶的資料集進行微調。
如此一來，便降低了 AI 部署生產品質所需的時間和成本，開發人員將能部署更龐大、更複雜的模型，以解決邊緣 AI 和機器人在自然語言理解、3D 感知及多感測器融合等方面所面臨的挑戰。

**新聞心得:**
難以想像一個只有手掌大小的超級電腦，聽起來就很迷你但是感覺它的功能卻是超乎那個大小可以做的事!而且每秒可以進行200兆次的運算，這個計算的運算量真的有夠大的!

**新聞來源:**
https://technews.tw/2021/11/10/jetson-agx-orin/
![圖_NVIDIA-推出全新-Jetson-AGX-Orin-機器人電腦，為未來的邊緣人工智慧與自主機器開拓發展之路-624x351](https://user-images.githubusercontent.com/90883526/141122781-7cb6924f-0c8f-4f42-bd4e-6d7c219ff397.jpg)

**Comment by 110409538** [comment]
這個超級電腦也太可愛了吧！

**Comment by minniewei** [comment]
這麼小小的電腦竟然有那麼強大的功能，科技果然進步得很快速呢!!

**Comment by Shiba010** [comment]
會不會在未來更進步之後，一個電腦可以做成一顆骰子大小？！

**Comment by ghost** [comment]
好像來到小人國的概念 哈哈
